generate_templates.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+command -v ruby >/dev/null 2>&1 || { echo "ERROR: Please install Ruby." >&2; exit 1; }
```

Comment:
It might be easier to use a "test" here instead ("man test") rather than have all the redirects.